src/sv-pipeline/04_variant_resolution/scripts/StitchFragmentedCNVs.java

```diff
@@ -0,0 +1,260 @@
+import java.io.BufferedOutputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.*;
+import java.util.zip.GZIPOutputStream;
+
+/** Read a VCF, and try to stitch together adjacent copy-number variations.
+ * Eligible Records (which we call "stitchable") must meet certain criteria as specified by the
+ * isStitchable method of the StitchableIterator.
+ * If two stitchables overlap appropriately, and all their samples have identical genotypes, we can
+ * replace the first one by adding on the interval covered by the second one.
+ */
+public class StitchFragmentedCNVs {
+    private static final VCFParser.ByteSequence END = new VCFParser.ByteSequence("END");
+    private static final VCFParser.ByteSequence SVLEN = new VCFParser.ByteSequence("SVLEN");
+    private static final double PAD_FACTOR = .2;
```

Comment:
It may have been intended that way, but the EXTEND parameter in the original script was completely unused.  Nonetheless, it was well documented as to intent, so I've separated the two params as requested.